corehq/pillows/mappings/case_mapping.py

```diff
@@ -41,7 +41,8 @@
                     },
                     'type': 'nested'},
         'closed': {'type': 'boolean'},
-        'closed_by': {'type': 'string'},
+        # todo; check with team
```

Comment:
Do these need to be addressed before the PR is merged?

Reply:
>  I am not updating the mapping as a reindex is not necessary.

This is what I wanted to check is okay. I guess that's okay, I will remove the todo comments.

Reply:
This is good to go?

Reply:
It is still not clear if you verified that the comments needed to be addressed or not. :+1: from me if they are no longer important.